scenarios/pfcp-agent-failure.xml

```diff
@@ -24,14 +27,25 @@
         <step name="AF-Check-Flows-2" requires="^"
               exec="onos-cli-grep ${OCI} up4:read-flows 'UL flows=0, DL flows=0'"/>
 
+        <step name="AF-Mid-Check-Interfaces" requires="^"
+              exec="onos-cmd ${OCI} 'up4:read-entities -i -g' 'tmp/Mid_check_interfaces.log'"/>
+
         <!-- Reboot SMF because it probably doesn't react well to failures -->
         <group name="AF-Restart-Smf" requires="~AF-Check-Flows-2">
-            <step name="AF-Kill-Smf"
-                  exec="mock-smf-cmd kill"/>
-            <step name="AF-Start-Smf" requires="^"
-                  exec="mock-smf-cmd start"/>
-            <step name="AF-Associate-Smf" requires="^" delay="1"
-                  exec="mock-smf-cmd associate"/>
+            <step name="AF-Kill-Smf" delay="1"
```

Comment:
Do we need the delay or was only for tests?

Reply:
It was there for test. No need to delay that step.